viz.yaml

```diff
@@ -270,7 +276,9 @@ publish:
     depends:
       header: header
       footer: footer
+	  intro: intro
       map-section: map-section
+	  intro2: intro
```

Comment:
i dont know, do those need unique IDs? I will call it Intro if not. Its the intro to the 2nd part....